zhaquirks/konke/button.py

```diff
@@ -112,3 +123,79 @@ def handle_message(self, profile, cluster, src_ep, dst_ep, message):
             },
         },
     }
+
+    device_automation_triggers = {
+        (DOUBLE_PRESS, DOUBLE_PRESS): {COMMAND: COMMAND_DOUBLE},
+        (SHORT_PRESS, SHORT_PRESS): {COMMAND: COMMAND_SINGLE},
+        (LONG_PRESS, LONG_PRESS): {COMMAND: COMMAND_HOLD},
+    }
+
+
+class KonkeButtonRemote2(CustomDevice):
```

Comment:
Let’s make a KonkeButton class that only has handle_message and extends CustomDevice. Then have both quirk device classes extend this so we don’t have 2 copies of handle_message. Ping me on discord if this isn’t clear.

Reply:
I may have gone too far, this is tested and worked for me, and sort of follows what was done in the Tuya remotes.